dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/canvas/taskbar.scss

```diff
@@ -174,8 +174,8 @@
             &.icos-switch {
               background-image: url("../images/task-icos/switch_hover.png");
             }
-            &.icos-tis {
-              background-image: url("../images/task-icos/tis_hover.png");
+            &.icos-pigeon {
```

Comment:
Why we have two `tis` icos here in L159 and L177

Reply:
add duplicate, should be reserved one

Reply:
has done

Reply:
> add duplicate, should be reserved one

done